telethon/tl/session.py

```diff
@@ -109,15 +112,15 @@ def list_sessions():
                 for f in os.listdir('.') if f.endswith('.session')]
 
     @staticmethod
-    def try_load_or_create_new(session_user_id):
+    def try_load_or_create_new(session_user_id, use_v6=False):
         """Loads a saved session_user_id.session or creates a new one.
            If session_user_id=None, later .save()'s will have no effect.
         """
         if session_user_id is None:
-            return Session(None)
+            return Session(None, use_v6)
```

Comment:
I'm not too sure about this, having to pass `use_v6` around everywhere. I will change this to have `None` IP address, and if this is the case, get the right IP from somewhere else.